content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Synchronized file sharing
+description: Get started with Synchronized file sharing on Docker Desktop.
+sitemap: false
+---
+
+> **Early Access**
+>
+> Synchronized file sharing is an early-access feature. 
+>
+> If you want to test out Synchronized file sharing, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{ .restricted }
+
+Synchronized file sharing is an alternative file sharing mechanism powered by [Mutagen](https://mutagen.io/). It provides fast and flexible host-to-VM file sharing by replacing bind mounts with synchronized filesystem caches. 
+
+![Image of Synchronized file shares pane](images/synched-file-sharing.png)
+ 
+## Who is it for?
+
+For developers who: 
+- Have large repositories or monorepos with 100 000 files or more totaling hundreds of megabytes or even gigabytes
+- Are using virtual filesystems, such as VirtioFS, gRPC FUSE, and osxfs, which are no longer scaling well with their codebases 
+- Regularly encounter performance limitations
+- Don't want to worry about file ownership or spend time resolving conflicting file-ownership information when modifying multiple containers
+
+## How does Synchronized file sharing work?
```

Comment:
This section could be better. Not sure how technical we want to get. 
Maybe an addition of a diagram?

Reply:
I think a diagram would be very helpful, esp. something that's comparative with the standard Virtiofs/gRPC-FUSE architecture.  I don't know if Docker has any baseline resources on hand that we could use.  I don't know if this is critical at EAP stage though; we can probably get along without it for now.

Reply:
Ok great, will make a note to do this post-EAP